Solution/HW08_Task1/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace HW08_Task1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите четверостишие:");
+            string poem = Console.ReadLine();
+
+            if (poem =="")
```

Comment:
we have learned that with strings we are need to use Equals with Comparison Type parameter instead of "==" operator. 
Just keep in mind!